analyses/doublet-detection/scripts/00b_format-benchmark-data.R

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env Rscript
+
+# This script converts the datasets used for benchmarking into SCE and AnnData objects,
+# mirroring ScPCA dataset format.
+# The original data files were obtained from Zenodo and are named <dataset name>.rds.
+# Each is an RDS object containing a list of two items:
+#  [[1]] A raw counts matrix
+#  [[2]] A vector of doublet/singlet calls for each barcode
+# The exported files are named `<dataset name>-<sce/anndata>.<rds/h5ad>.
+# Each has a variable `ground_truth_doublets` representing the singlet/doublet calls:
+#   - In the SCE file, this is in the colData slot
+#   - In the AnnData file, this is in the obs slot
+
+# Load renv
+renv::load("../")
```

Comment:
If you run this script from anywhere else this will break... I would define the project root and then load that.

Reply:
```suggestion
project_root <- here::here()
renv::load(project_root)

```